docs/Agent/user-guide/man_page.md

```diff
@@ -450,44 +543,51 @@ Invoking `pbench-user-benchmark` with a workload generator as an argument will p
 - Stop the collection tools on all the hosts.
 - Gather the data from all the remote hosts and generate a `result.txt` file by running the tools' post-processing on the collected data.
 
-`<command-to-run>`
+`<command-to-run>`\
 A script, executable, or shell command to run while gathering tool data. Use `--`
 to stop processing of `pbench-user-benchmark` options if your command includes
 options, like
 
 > `pbench-user-benchmark --config string -- fio --bs 16k`
 
-**OPTIONS**  
-`-C`, `--config PATH`  
+**OPTIONS**
+
+[`-C`, `--config`] `<path>`\
 Path to the Pbench Agent configuration file.
 This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
 
-`--tool-group STR`  
+`--tool-group <tool-group>`\
 The tool group to use for data collection.
 
-`--iteration-list STR`  
-A file containing a list of iterations to run for the provided script;
-the file should contain one iteration per line. With a leading hash (`#`) character used for comments and blank lines are ignored.
-Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script.  
+`--iteration-list <file>`\
+A file containing a list of iterations to run for the provided script. The file
+must contain one iteration per line. Blank lines are ignored, and you can use a
+leading hash (`#`) character for comments. Each iteration line should use
+alpha-numeric characters before the first space to name the iteration, with the
+rest of the line provided as arguments to the script.
 _NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
-`--sysinfo STR[,STR...]`  
-Comma-separated values of system information to be collected; available: `default`, `none`, `all`, `ara`, `block`, `insights`, `kernel_config`, `libvirt`, `security_mitigations`, `sos`, `stockpile`, `topology`
+`--sysinfo STR[,STR]...`\
+Comma-separated values of system information to be collected; available:
+`default`, `none`, `all`, `ara`, `block`, `insights`, `kernel_config`,
+`libvirt`, `security_mitigations`, `sos`, `stockpile`, `topology`
```

Comment:
All the other option value specifiers are semantic mnemonics in angle brackets...should we do the same, here?  (We can use `<option>` if you cannot come up with something better.)

Also, I'm not crazy about the telegraphic style embodied in `available`.  I think we can afford to have a full sentence, like, "the available options are:".

Reply:
I clearly just missed this one when I was skimming through. 😦